### PR TITLE
Changes to md file for the app

### PR DESCRIPTION
**Pull request by OverPoweredDev**
Fixes Issue #12 

_Note: I've commented out the parts of the shields with downloads and rating, so just put those back in when it gains more traction_

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,66 @@
...
+<img src="https://lh3.googleusercontent.com/KiRgfZoPqX41YJZsexeNWcq6BS80cvDtx7LJ387bHL9dJq7AfTveJdoHFJTCROF9QA=w1536-h759-rw" alt="Screenshot2" width="320"/>
+
+
+## Development
+ The app makes use of the following technologies:
+ 
+ * Kotlin over traditional Java in Android Development
+ * App makes use of the new and improved `AndroidX` libraries in place of the `Support Libraries`
+ * Firebase Realtime Database
+ * Firebase UI (Uses Firebase Recycler Adapter)
```
Comment (Devansh-Maurya) [inline README.md]: We are not using Firebase Recycler Adapter. Also, we are using Firestore, not Realtime database.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,66 @@
...
+ * Firebase Cloud Messaging (FCM)
+ * Lottie (for displaying animations in intro slider)
+ * Shape of View (to modify view shapes)
+ * Toast Splitties (Kotlin Extension)
+ * Timber (extensible to provide utility over Android's Log class)
+
+And a couple more!
```
Comment (Devansh-Maurya) [inline README.md:59]: You can add more info. like architecture patterns and other Jetpack components. Please refer [this repo](https://github.com/Devansh-Maurya/PukaPuka#technologies-used) or [this one](https://github.com/Devansh-Maurya/TreeCare#technologies-used)

**Review by Devansh-Maurya — CHANGES_REQUESTED** [review]

**Comment by Devansh-Maurya** [comment]
@golupankajgoyal Please provide more screenshots to be added.

File: README.md (hunk 2 above)
Comment (OverPoweredDev) [reply README.md:59]: Okay I'll fix this along with the other one

**Comment by OverPoweredDev** [comment]
Look Better?

**Comment by Devansh-Maurya** [comment]
Looks better now! Thanks for putting the effort to update the README. Will merge the PR now!

File: README.md (hunk 2 above)
Comment (Devansh-Maurya) [reply README.md:59]: Thanks

**Pull request merged**